packages/shared/src/components/ProfileMenu/ProfileSectionItem.tsx

```diff
@@ -60,16 +64,20 @@ export const ProfileSectionItem = ({
         color={TypographyColor.Tertiary}
         type={TypographyType.Subhead}
         className={classNames(
-          'flex cursor-pointer items-center gap-2 rounded-10 px-1 py-1.5 hover:bg-theme-active',
+          'flex cursor-pointer items-center gap-2 rounded-10 px-1 py-1.5',
+          (href || onClick) && 'hover:bg-surface-float',
+          isActive ? 'bg-surface-active' : undefined,
           className,
+          // TODO: remove this when all links are in place
+          !href && !onClick && '!cursor-default !text-text-disabled',
         )}
         {...combinedClicks(() => onClick?.())}
         {...(isExternal && { target: '_blank', rel: anchorDefaultRel })}
       >
         {icon}
         <span>{title}</span>
 
-        {isExternal && (
+        {(isExternal || showExternalIcon) && (
```

Comment:
Can we somehow merge this, sounds unclear what is what with multiple props.

Reply:
I refactored it a bit, now it shows icon if `external === true`, and then if `href` does not begins with `webappUrl`, it will open in new tab.